scripts/py_matter_idl/examples/matter_idl_plugin/matter_cluster_proto.jinja

```diff
@@ -0,0 +1,85 @@
+
+/// AUTO-GENERATED with matter_idl.
+
+syntax = "proto3";
+
+package com.chip.proto;
+
+option java_multiple_files = true;
+
+message {{cluster.name}}Trait {
+  // option (message_type) = MATTER_TRAIT;
+
+  enum ClusterId {
+    CLUSTER_ID_UNSUPPORTED = 0;
+    CLUSTER_ID = {{cluster.code}};
+  }
+
+  // Enums
```

Comment:
does this comment help? The list will already contain 'enum' and if the items are empty, the generated files will look like:

```
// Enums
// Bitmaps
// Structs
struct Foo {
...
```

which seems not that useful